scripts/update/01-run.sh

```diff
@@ -4,6 +4,9 @@ set -euo pipefail
 RELEASE=$1
 UMBREL_ROOT=$2
 
+# Only used on Umbrel OS
+SD_CARD_UMBREL_ROOT="/sd-root/home/umbrel/umbrel"
```

Comment:
Shouldn't we use `/sd-root${UMBREL_ROOT}` here?

Reply:
Ah, yes absolutely. Updated!